packages/gestalt/src/Text.css

```diff
@@ -1,3 +1,9 @@
 .Text {
   composes: antialiased sansSerif from "./Typography.css";
 }
+
+*:lang(zh) .Text,
```

Comment:
can we add a comment above explaining what this is preveinting?

Reply:
Same for Headings?

Reply:
Anywhere the .Text class is applied

Reply:
Let's test this on an alpha release actually!